react/src/util/opReturn.ts

```diff
@@ -55,10 +68,10 @@ function generatePushdataPrefixedPaymentId(bytesAmount: number): string {
   return prependPaymentIdWithPushdata(hexString);
 }
 
-function getDataPushdata(data: string, disablePaymentId=false) {
+function getDataPushdata(data: string, disablePaymentId = false) {
   const bytesQuantity = new Blob([data]).size;
-  // If paymentId is expected, limit is 9 bytes smaller
-  const bytesLimit = USER_DATA_BYTES_LIMIT - (disablePaymentId ? 0 : 9)
+  // If nonce is expected, limit is 8 bytes smaller
```

Comment:
Intentionally changed this from `paymentID` to `nonce`?

Reply:
Indeed this was a mistake, I've just changed (here and where else it had 'nonce')